data/json/monsterdrops/feral_humans.json

```diff
@@ -82,5 +82,23 @@
       { "group": "clothing_watch", "prob": 5 },
       { "group": "wallets", "damage": [ 1, 4 ] }
     ]
+  },
+  {
+    "type": "item_group",
+    "subtype": "collection",
+    "id": "starer_death_drops_axe",
+    "entries": [ { "item": "fire_ax", "prob": 100, "damage": [ 2, 4 ] } ]
```

Comment:
Are partially burnt items spawnable in itemgroups?

Reply:
Ooh, I like that a lot.

Reply:
But no, not yet as far as I can tell